CODE_OF_CONDUCT.md

```diff
@@ -1,9 +0,0 @@
-# Reglas básicas
```

Comment:
¿Por qué borraste el código de conducta? También noté que borraste los .keep de `common/, `docs/`y `img/`y el `.editorconfig` (que estaban en desuso, pero intencionalmente)

Reply:
El código de conducta lo puse en el `readme.md` y los otros archivos que estaban vacíos los borre porque hacían que podían llegar a confundir. Cuando sean necesarios los deberíamos crear, pero por ahora solo estaban ahí por estar.

Reply:
Ya restauré el código de conducta.